docs/recipes/ec2-eks-metrics-go-adot-ampamg.md

```diff
@@ -1,12 +1,314 @@
+
+The following template file uses a deamonset for it's configuration [prometheus-daemonset.yaml](ec2-eks-metrics-go-adot-ampamg/prometheus-daemonset.yaml).
+
+AWS Fargate does not support deamonsets so you can use the following template [prometheus-fargate.yaml](ec2-eks-metrics-go-adot-ampamg/prometheus-fargate.yaml), if your cluster was setup with AWS Fargate.
+
+
+#### Edit your template file
+
+In this example, the ADOT Collector configuration uses an annotation (scrape=true) to tell which target endpoints to scrape. This allows the ADOT Collector to distinguish the sample app endpoint from kube-system endpoints in your cluster. You can remove this from the re-label configurations if you want to scrape a different sample app. 
+
+Use the following steps to edit the downloaded file for your environment:
+
+1\. Replace **<REGION\>** with your current Region. 
+
+2\. Replace **<YOUR_ENDPOINT\>**  with your AMP workspace endpoint URL.
+
+Get your AMP endpoint url by executing the following query:
+```
+$aws amp describe-workspace \ 
+    --workspace-id `aws amp list-workspaces --alias prometheus-sample-app --query 'workspaces[0].workspaceId' --output text` \
+    --query 'workspace.prometheusEndpoint'
+```
+
+3\. Finally replace your **<YOUR_ACCOUNT_ID\>**  with your current account ID.
+
+The following command will return the account ID for the current session:
+```
+aws sts get-caller-identity --query Account --output text
+```
+<br/>
+
+#### Deploy the template to your cluster
+
+```
+$ kubectl apply -f prometheus-fargate.yaml
+```
+
+In case you used the deamonset template use the following command
+```
+$ kubectl apply -f prometheus-daemonset.yaml
+```
+
+You can verify that the ADOT Collector has started with this command:
+
+```
+$ kubectl get pods -n adot-col
+```
+
+For more information check out the [AWS Distro for OpenTelemetry (ADOT) Collector Setup](https://aws-otel.github.io/docs/getting-started/prometheus-remote-write-exporter/eks#aws-distro-for-opentelemetry-adot-collector-setup)
+
+### Setup AMG
+
+[Amazon Managed Service for Grafana – Getting Started](https://aws.amazon.com/blogs/mt/amazon-managed-grafana-getting-started/)
+
+Setup a new AMG workspace using the Getting Started guide above.
+
+Make sure to add "Amazon Managed Service for Prometheus" as a datasource during creation.
+
+![Service managed permission settings](https://d2908q01vomqb2.cloudfront.net/972a67c48192728a34979d9a35164c1295401b71/2020/12/09/image008-1024x870.jpg)
+
+
 ## Application
 
+The following sample application will be used in this guide:
+
+[Sample App Github](https://github.com/aws-observability/aws-otel-community/tree/master/sample-apps/prometheus)
+
+
+### Build
+Clone the following Git repository
+```
+$ git clone git@github.com:aws-observability/aws-otel-community.git
+```
+
+Build the container
+```
+$ cd ./aws-otel-community/sample-apps/prometheus
+$ docker build . -t prometheus-sample-app:latest
```

Comment:
It is a good idea to add a note for "go mod fails proxy.golang.or i/o timeout" due security restrictions on laptops.  change Dockerfile 
# RUN GO111MODULE=on go mod download
RUN GOPROXY=direct GO111MODULE=on go mod download